EventFilter/CTPPSRawToDigi/python/totemRawToDigi_cff.py

```diff
@@ -1,7 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
 from CondFormats.CTPPSReadoutObjects.TotemDAQMappingESSourceXML_cfi import *
-TotemDAQMappingESSourceXML.mappingFileNames.append("CondFormats/CTPPSReadoutObjects/xml/ctpps_210_mapping.xml")
+totemDAQMappingESSourceXML.configuration = cms.VPSet(
+  # before TS2 (2016)
+  cms.PSet(
+    validityRange = cms.EventRange("1:min - 280385:max"),
+    mappingFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/ctpps_mapping_to_fill_5288.xml"),
+    maskFileNames = cms.vstring()
+  ),
+  # during TS2 (2016)
+  cms.PSet(
+    validityRange = cms.EventRange("280386:min - 281600:max"),
+    mappingFileNames = cms.vstring(),
```

Comment:
we have some data in Run2016H-v1 in this range of runs in physics PDs.
If some reprocessing ends up triggered for it (privately or centrally), it should be able to run.